CRM/Cplabels/Form/Search/Cplabels/Client.php

```diff
@@ -125,18 +125,15 @@ public function buildForm(&$form) {
      */
     $form->assign('elements', array('team', 'diagnosis', 'min_age', 'max_age', 'limit'));
 
-    $form->addRule('team', E::ts('Field "Team" is required'),'required');
-    $form->addRule('diagnosis', E::ts('Field "Diagnosis" is required'),'required');
+    $form->addRule('team', E::ts('Field "Team" is required'), 'required');
+    $form->addRule('diagnosis', E::ts('Field "Diagnosis" is required'), 'required');
   }
 
   /**
    * Get a list of summary data points
    *
-   * @return mixed; NULL or array with keys:
-   *  - summary: string
-   *  - total: numeric
```

Comment:
I'd like to keep the @return documentation in these lines. The lint error for these two lines is "Return comment indentation must be 3 spaces, found 2 spaces". So I think you can fix this lint error by simply adjusting the space at the front of these two lines (136 and 137).

Reply:
Fixed